src/software/thunderscope/robot_communication.py

```diff
@@ -71,5 +183,19 @@ def __enter__(self):
             self.multicast_channel + "%" + self.interface, VISION_PORT, True
         )
 
+        self.connect_fullsystem_to_robots()
+
+        # TODO-NIMA-ARUN we might not want to support robot diagnostics in tscope
+        # make a ticket here to create a widget to call these functions to detach
+        # from AI and connect to robots/or remove
+        # self.disconnect_fullsystem_from_robots()
+        # self.connect_robot_to_diagnostics(0)
```

Comment:
Thanks for noting this. I've created #2711 for now. I'm not sure if I fully understood the task and explained it properly. Feel free to update it.
Also, curious when `self.disconnect_fullsystem_from_robots()` should be used and if its necessary?